manuscript/chapter5.md

```diff
@@ -139,7 +139,7 @@ const Search = ({
         value={value}
         onChange={onChange}
 # leanpub-start-insert
-        ref={(node) => input = node}
+        ref={el => this.input = el}
```

Comment:
Does `el` make sense for everyone? I know the React documentation uses it as well, but maybe `element` is more specific.

Reply:
Ooops, I will revert it

Reply:
So with the right alignment, does it really look better than the center? https://www.fotor.com/share/92ed452d-bbe0-4ba0-b3ab-c98580061483

Reply:
I think it's better to do so that there is no difference with the documentation?

Reply:
Hm okay. Does it say "element" somewhere in the text? If not, then maybe add it over there.

Reply:
Replace 

> The `this` object of an ES6 class component helps us to reference the DOM node with the `ref` attribute.

with

> The `this` object of an ES6 class component helps us to reference the DOM __element__ with the `ref` attribute.

Right?

Reply:
Yep. Then it's alright 👍

Reply:
Done :ballot_box_with_check: